docs/azure/hosted-control-plane.md

```diff
@@ -0,0 +1,162 @@
+  config:
+    location: "westus"
+    subscriptionID: ceb131c7-a917-439f-8e19-cd59fe247e03
+    vmSize: Standard_A4_v2
+    clusterIdentity:
+      name: az-cluster-identity
+      namespace: hmc-system
+    resourceGroup: mgmt-cluster
+    network:
+      vnetName: mgmt-cluster-vnet
+      nodeSubnetName: mgmt-cluster-node-subnet
+      routeTableName: mgmt-cluster-node-routetable
+      securityGroupName: mgmt-cluster-node-nsg
+    tenantID: 7db9e0f2-c88a-4116-a373-9c8b6cc9d5eb
+    clientID: 471f65fa-ddee-40b4-90ae-da1a8a114ee1
+    clientSecret: "u_RANDOM"
+```
+
+To simplify creation of the deployment object you can use the template below:
+
+```yaml
+apiVersion: hmc.mirantis.com/v1alpha1
+kind: Deployment
+metadata:
+  name: azure-hosted-cp
+spec:
+  template: azure-hosted-cp
+  config:
+    location: "{{.spec.location}}"
+    subscriptionID: "{{.spec.subscriptionID}}"
+    vmSize: Standard_A4_v2
+    clusterIdentity:
+      name: az-cluster-identity
+      namespace: hmc-system
+    resourceGroup: "{{.spec.resourceGroup}}"
+    network:
+      vnetName: "{{.spec.networkSpec.vnet.name}}"
+      nodeSubnetName: "{{(index .spec.networkSpec.subnets 1).name}}"
+      routeTableName: "{{(index .spec.networkSpec.subnets 1).routeTable.name}}"
+      securityGroupName: "{{(index .spec.networkSpec.subnets 1).securityGroup.name}}"
+    tenantID: 7db9e0f2-c88a-4116-a373-9c8b6cc9d5eb
+    clientID: 471f65fa-ddee-40b4-90ae-da1a8a114ee1
+    clientSecret: "u_RANDOM"
+```
+
+Then you can render it using the command:
+
+```bash
+kubectl get azurecluster <management cluster name> -o go-template="$(cat template.yaml)"
+```
+
+## Cluster creation
+
+After applying `Deployment` object you require to manually set the status of the
+`AzureCluster` object due to current limitations (see k0sproject/k0smotron#668).
+
+To do so you need to execute the following command:
+
+```bash
+kubectl patch azurecluster <cluster name> --type=merge --subresource status --patch 'status: {ready: true}'
+```
+
+## Important notes on the cluster deletion
+
+Because of the aforementioned limitation you also need to make manual steps in
+order to properly delete cluster.
+
+Before removing the cluster make sure to place custom finalizer onto
+`AzureCluster` object. This is needed to prevent it from being deleted instantly
+which will cause cluster deletion to stuck indefinitely.
+
+To place finalizer you can execute the following command:
+
+```bash
+kubectl patch azurecluster <cluster name> --type=merge --patch 'metadata: {finalizers: [manual]}'
+```
+
+When finalizer is placed you can remove the `Deployment` as usual. Check that
+all `AzureMachines` objects are deleted successfully and remove finalizer you've
+placed to finish cluster deletion.
+
+In case if have orphaned `AzureMachines` left you have to delete finalizers on
+them manually after making sure that no VMs are present in Azure.
+
+*Note: since Azure admission prohibits orfaned objects mutation you'll have to
```

Comment:
`orphaned`

Reply:
@eromanova I alredy fixed it [here](https://github.com/Mirantis/hmc/compare/0d9c1426a20849f3e3eea50d4241f199a4706802..73dd0f0bc90cb9f2c7688a178c936fdc022a0112). Somehow you see outdated version

Reply:
Correct. Thanks.

Reply:
Oh, no. The second typo` is on the 161 line. :)

Reply:
Well, shit))

Reply:
Now it's fixed for good (I hope)